README.md

```diff
@@ -116,6 +116,7 @@
 * [**Go**: _Building a cli command with Go: cowsay_](https://flaviocopes.com/go-tutorial-cowsay/)
 * [**Go**: _Go CLI tutorial: fortune clone_](https://flaviocopes.com/go-tutorial-fortune/)
 * [**Nim**: _Writing a stow alternative to manage dotfiles_](https://xmonader.github.io/nimdays/day06_nistow.html)
+* [**Rust**: Writing a wget clone in Rust](https://mattgathu.github.io/writing-cli-app-rust/)
```

Comment:
Change the title to:

> Writing a wget clone

It is already noted that the tutorial is for rust.

Reply:
i changed it.